src/services/jwt.service.ts

```diff
@@ -0,0 +1,15 @@
+import jwt from 'jsonwebtoken';
+import { User } from '@entities/user.entity';
+import { JWT_SECRET } from '@config';
+
+export const createJWT = (user: User) => {
+  return jwt.sign(
+    { data: { userId: user.id, email: user.email } },
```

Comment:
Should we include the hashed password? That way when the user changes his/her password it would make JWT invalid 🤔

Reply:
Makes sense...

Reply:
i think that in the case the user changes the password you have to store the old token in a "revoked token list" in the database, or even better in something like Redis, as long as the old token gets expired by itself

Reply:
Yes. I added as a note in the PR's description. I like both approaches. We can discuss which one better suits us

Reply:
Related to this https://github.com/rootstrap/express-api-base/pull/6#discussion_r492681468